operations/validate_csm_health.md

```diff
@@ -717,188 +713,51 @@ beyond the dracut stage of the boot process. However, if the dracut stage is rea
 boot can be considered successful and shows that the necessary CSM services needed to
 boot a node are up and available.
    * This inability to boot the barebones image fully will be resolved in future releases of the
-   CSM product.
+CSM product.
 * In addition to the CSM Barebones image, the release also includes an IMS Recipe that
 can be used to build the CSM Barebones image. However, the CSM Barebones recipe currently requires
 RPMs that are not installed with the CSM product. The CSM Barebones recipe can be built after the
 Cray OS (COS) product stream is also installed on to the system.
    * In future releases of the CSM product, work will be undertaken to resolve these dependency issues.
 * This procedure can be followed on any NCN or the PIT node.
-* The Cray CLI must be configured on the node where this procedure is being performed. See [Configure the Cray Command Line Interface](configure_cray_cli.md) for details on how to do this.
+* This script uses the kubernetes API Gateway to access CSM services.  This gateway must be properly
```

Comment:
Nits: Kubernetes should be capitalized, and the standard in the docs is single spacing between sentences.